src/assets/wise5/classroomMonitor/dataExport/strategies/AbstractDataExportStrategy.ts

```diff
@@ -4,15 +4,14 @@ import { ConfigService } from '../../../services/configService';
 import { DataExportService } from '../../../services/dataExportService';
 import { TeacherDataService } from '../../../services/teacherDataService';
 import { TeacherProjectService } from '../../../services/teacherProjectService';
-import { DataExportComponent } from '../data-export/data-export.component';
 import { DataExportContext } from '../DataExportContext';
 import { DataExportStrategy } from './DataExportStrategy';
 import { removeHTMLTags } from '../../../common/string/string';
 import { generateCSVFile } from '../../../common/csv/csv';
 
 export abstract class AbstractDataExportStrategy implements DataExportStrategy {
   context: DataExportContext;
-  controller: DataExportComponent;
+  controller: any;
```

Comment:
Why are we relaxing the typing of the controller from DataExportComponent to any here?

Reply:
This is due to the tight coupling between the strategies and the controller which I don't think should have been implemented like this in the first place. More refactoring in the export strategies will need to be done in the future to reduce the coupling.

Reply:
I see. In this case, should we use a [union type](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#union-types) so we don't allow any type?